tools/integration_tests/write_large_files/seq_write_large_file_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package write_large_files
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/operations"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup"
+)
+
+const (
+	FiveHundredMB                             = 500 * OneMiB
+	FiveHundredMBFile                         = "fiveHundredMBFile.txt"
+	ChunkSize                                 = 20 * OneMiB
+	DirForSeqWrite                            = "dirForSeqWrite"
+	FiveHundredMBFileForSeqWriteInLocalSystem = "fiveHundredMBFileForSeqWriteInLocalSystem"
+)
+
+func TestWriteLargeFileSequentially(t *testing.T) {
+	seqWriteDir := path.Join(setup.MntDir(), DirForSeqWrite)
+	err := os.Mkdir(seqWriteDir, setup.FilePermission_0600)
+	if err != nil {
+		t.Fatalf("Error in creating directory:%v", err)
+	}
+	filePath := path.Join(seqWriteDir, FiveHundredMBFile)
+
+	// Sequentially read the data from file.
+	err = operations.WriteFileSequentially(filePath, FiveHundredMB, ChunkSize)
+	if err != nil {
+		t.Fatalf("Error in writing file: %v", err)
+	}
+
+	// Clean up.
+	defer operations.RemoveDir(seqWriteDir)
```

Comment:
Move this before `operations.WriteFileSequentially(filePath, FiveHundredMB, ChunkSize)` so that even if the file-write fails, this directory gets deleted.